apps/settings/js/panels/apn_editor/panel.js

```diff
@@ -92,8 +92,7 @@ define(function(require) {
               })
             ]).then(function(results) {
               var activeApnId = results[0];
-              var dataRoamingEnabled = results[1];
-              if (activeApnId === _apnItem.id && dataRoamingEnabled) {
+              if (activeApnId === _apnItem.id) {
```

Comment:
we can remove above `Promise.all` and `new Promise((resolve) => {}` and use `new Promise` instead  since the `dataRoamingEnabled` is not used. 

The comment ```
// Display the warning only when Data roaming is turned on and it is
// the current APN in use that’s being edited.
```
should be updated as well